CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.0.0](https://github.com/merkle-open/env-linter/compare/v0.5.1...v0.6.0) (2022-04-29)
```

Comment:
compare v0.5.1 with v1.0.0

Reply:
> There is nothing else to say against a release 1.0. Of course there are again some dev dependencies that could be renewed, as well as the default node version, but that can always be done.
> 
> You could still run prettier (results in adjustments in the changelog)

I see your point of view. These package version change quite frequently and it is sometimes hard to keep track of... Nonetheless, the default node version in the .node-version file can be updated for sure.